README.md

```diff
@@ -72,6 +72,28 @@ Cookies.remove('name', { path: '/' }); // => true
 
 *Note: when deleting a cookie, you must pass the exact same path, domain and secure options that were used to set the cookie, unless you're relying on the default options that is.*
 
+## JSON
+
+js-cookie provides automatic JSON storage for cookies.
+
+When creating a cookie you can pass an Array or Object Literal instead of a string in the value. If you do so, js-cookie store the string representation of the object according to the `JSON.stringify` api (if available):
```

Comment:
We could document that if it's not a valid JSON structure then it will use `String(object)` in the input, but then we would be enforcing behavior for invalid input, and any changes that does something else would be backwards incompatible.

Assuming we are using [semver spec](semver.org), backwards incompatible changes should only be  release in major version (in this case v3.0).